rosbag2_transport/src/rosbag2_transport/topic_filter.hpp

```diff
@@ -56,6 +58,12 @@ filter_topics_with_known_type(
   const std::unordered_map<std::string, std::string> & topics_and_types,
   std::unordered_set<std::string> & topic_unknown_types);
 
+ROSBAG2_TRANSPORT_PUBLIC
+std::unordered_map<std::string, std::string>
+filter_leaf_topics(
```

Comment:
Can you add a docstring for this please? I know the other functions don't have them (but that's a bad and confusing thing)

Reply:
Sounds good, fixed in https://github.com/ros2/rosbag2/pull/925/commits/da80c562101222cfb97583369de5811cb50e38be.